.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     - id: end-of-file-fixer
     - id: trailing-whitespace
  - repo: https://github.com/lycheeverse/lychee
-   rev: cef607cddb6279091869bcbb8fb4b906fd4c046c  # frozen: nightly
+   rev: 8052c78a3fc7f74b6b987264aece2ac2f4f9a044  # frozen: nightly
```

Comment:
Blocked on https://github.com/lycheeverse/lychee/issues/1833 / https://github.com/lycheeverse/lychee/pull/1834

Reply:
Workaround in place